x-pack/packages/kbn-elastic-assistant-common/env/http-client.env.json

```diff
@@ -4,11 +4,22 @@
     "port": "5601",
     "basePath": "",
     "elasticApiVersion": "1",
+    "elasticsearch": {
+      "host": "localhost",
+      "port": "9200"
+    },
     "auth": {
-      "username": "elastic",
-      "password": "changeme"
+      "admin": {
+        "username": "elastic",
+        "password": "changeme"
+      },
+      "assistant_system": {
```

Comment:
nit: consider a different name for this test user, because there's no "system" account required at runtime for this feature

Reply:
Will rename to `Assistant All` 👍

Reply:
Fixed in https://github.com/elastic/kibana/commit/ac494d7767a9a1acb53c6f7bd4143b271f15fb49